src/components/BookList.js

```diff
@@ -1,75 +1,19 @@
-import { useState } from 'react';
-import { v4 as uuidv4 } from 'uuid';
+import { useSelector } from 'react-redux';
 import Book from './Book';
 import AddForm from './AddForm';
 
 const BookList = () => {
-  const [books, setBooks] = useState([
-    {
-      id: uuidv4(),
-      title: 'Book 1',
-      author: 'Author 1',
-      progress: '60%',
-    },
-    {
-      id: uuidv4(),
-      title: 'Book 2',
-      author: 'Author 2',
-      progress: '76%',
-    },
-  ]);
-  const [title, setTitle] = useState('');
-  const [author, setAuthor] = useState('');
-
-  const handleDelete = (id) => {
-    const newBooks = books.filter((book) => book.id !== id);
-    setBooks(newBooks);
-    return books;
-  };
-
-  const handleTitleChange = (e) => {
-    setTitle(e.target.value);
-  };
-
-  const handleAuthorChange = (e) => {
-    setAuthor(e.target.value);
-  };
-
-  const handleAddBook = (e) => {
-    e.preventDefault();
-    const newBook = {
-      id: uuidv4(),
-      title,
-      author,
-      progress: '0%',
-    };
-    setBooks([...books, newBook]);
-
-    setTitle('');
-    setAuthor('');
-  };
+  const books = useSelector((store) => store.books);
 
   return (
     <section>
       <div className="books-container">
         {books.map((book) => (
-          <Book
-            key={book.id}
-            id={book.id}
-            title={book.title}
-            author={book.author}
-            progress={book.progress}
-            handleDelete={handleDelete}
-          />
+          // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
- [ ] It is not best practice to disable linter on your codebase. Linter was put in place to help prevent errors and unaccepted syntax. Please remove this

Reply:
Hi @bobb-Rob, first of all, thanks for your time! You are right, I disable that linter because on the tutorial we saw the properties are sent to the child using {...item}. Now I fixed that and remove the line to disable the linter. 
Thank you again for your review! Have a nice day!